charts/newrelic-pixie/values.yaml

```diff
@@ -28,3 +28,6 @@ resources:
   requests:
     cpu: 100m
     memory: 250M
+
+proxy: {}
```

Comment:
Should this be an object (`{}`) or an string? From the `deployment.yaml` template I understand it should be a string, right? Or are we planning to split http and https proxies?

Reply:
According to the docs, both an empty string and empty object should work: https://helm.sh/docs/chart_template_guide/control_structures/

Reply:
We've seen helm complaining when switching types, for example overriding an empty object with an array, but in this case it doesn't seem to matter 😕 

In any case, if Helm is fine with it, so I am :)